anesthetic/plot.py

```diff
@@ -269,6 +269,9 @@ def fastkde_plot_1d(ax, data, *args, **kwargs):
 
     xmin = kwargs.pop('xmin', None)
     xmax = kwargs.pop('xmax', None)
+    cmap = kwargs.pop('cmap', None)
+    color = kwargs.pop('color', (next(ax._get_lines.prop_cycler)['color']
+                                 if cmap is None else cmap(2/3)))
```

Comment:
Probably safer for those used to reading python2 to make this 2./3., or 0.66 rather than 2/3 (since in python2, this would be 0). Better to be explicit (even though it doesn't matter for python3). Should be changed throughout.

Reply:
done